src/modules/web3/model/requested-speed-up-deposit-ev.entity.ts

```diff
@@ -0,0 +1,43 @@
+import { Column, CreateDateColumn, Entity, PrimaryGeneratedColumn, Unique, UpdateDateColumn } from "typeorm";
+
+export type RequestedSpeedUpDepositEventArgs = {
```

Comment:
NIT: Can't we get this from the typechain bindings in `@across-protocol/contracts-v2`?